src/main/java/seedu/address/commons/core/Messages.java

```diff
@@ -9,5 +9,6 @@
     public static final String MESSAGE_INVALID_COMMAND_FORMAT = "Invalid command format! \n%1$s";
     public static final String MESSAGE_INVALID_EATERY_DISPLAYED_INDEX = "The eatery index provided is invalid";
     public static final String MESSAGE_EATERIES_LISTED_OVERVIEW = "%1$d eateries listed!";
+    public static final String MESSAGE_EATERY_CLOSED = "The eatery is closed!";
```

Comment:
i think this isn't needed! kai qi has already added a similar variable with the same command in the CloseCommand class i.e. "MESSAGE_EATERY_ALREADY_CLOSED". i feel that implementation (?) makes more sense as the messages here seem for like application-wide messages and not messages specific to the eateries!